### ISSUE TITLE
Create Script Questions for the Download Project to CSV File Usability Testing

### ISSUE THREAD

**Issue by abbyrivard** [hackforla/tdm-calculator#1966]
### Dependencies
- [x] #1965 
- [x] #2142 

### Overview
We need to generate questions for the Download project to CSV file usability testing that will identify any pain points and potential recommendations with the current designs and functions of this process.

#### Details
We need to identify all the work to be done for the general usability testing on the downloading a project to a CSV file from the My Projects page so that we can establish timelines and coordinate the work.

### Action Items
- [x] Review previous usability test scripts
- [x] Write test questions/tasks
- [x] Submit test questions/tasks to PM for approval
- [x] Add questions/tasks link to resources section of #1967 

- [x] Release the dependencies on #1969 

#### Resources for creating this issue
 - Reference previous research script - [RP:004 Reset Button Testing Script](https://docs.google.com/spreadsheets/d/1BDUu0Bfvi0LeVTwF_Yy2p2ZTtw6y0e27qfILaSRcuk4/edit#gid=613097829)
- #1664 
- - [RP007: Download to CSV Research Plan](https://docs.google.com/document/d/1R9w25xmyFhJ0IKO12n4qzRzgmZg11j1qb7ivQCVoTEY/edit?tab=t.0)

#### Resources gathered during the completion of this issue
- [RP007: Interview Script](https://docs.google.com/spreadsheets/d/1L9C78gcxWMRleAh0hPUK84fkF5zMJ3XoNJdU9_hu26k/edit?gid=170080311#gid=170080311)

**Comment by mh-faraji** [comment]
@melissaperry09 I completed a draft of the [testing script](https://docs.google.com/spreadsheets/d/1L9C78gcxWMRleAh0hPUK84fkF5zMJ3XoNJdU9_hu26k/edit?gid=170080311#gid=170080311).

**Comment by melissaperry09** [comment]
@ExperimentsInHonesty - the UXR team is doing an internal review of this script document to identify any areas we need to update @mh-faraji's initial draft. This review will be completed by 26-02-2025.

**Comment by senakim18** [comment]
hi @melissaperry09, do we have an update on the review? Thank you!

**Comment by melissaperry09** [comment]
@senakim18 - While doing our UXR team review, we identified a potential bug related to the CSV modal file naming text field. I have created issue #2142 to document this and have added the ready for dev lead label, so the remaining labels can be properly identified. Because using the file naming text field is part of our download to CSV testing script, we will not be able to submit our script for approval until we receive an update on issue #2142.

**Comment by mh-faraji** [comment]
@melissaperry09, has the bug with the CSV modal file naming text field been resolved?

**Comment by ExperimentsInHonesty** [comment]
@mh-faraji Please put the answer that they gave you to your question, into a comment on this issue.  

>@melissaperry09, has the bug with the CSV modal file naming text field been resolved?

Then decide if the information needs to be added to the top of this issue and add it there.  And then hide comments that are no longer needed.  Then remove the `ready for reserarch lead` label.

**Comment by mh-faraji** [comment]
The bug has been fixed, and this script is ready for review.

**Comment by ExperimentsInHonesty** [comment]
@mh-faraji @melissaperry09 @abbyrivard this script looks good to me. What are the next steps for this issue, I don't know what to do next.

**Comment by melissaperry09** [comment]
@ExperimentsInHonesty - Once the script is approved, we close the issue and then start the prep work for issue #1969 to conduct the testing. I will add this approved script to the Wiki and other related issues and close this issue.
